linkerd/app/outbound/src/endpoint.rs

```diff
@@ -323,14 +331,35 @@ impl tls::HasPeerIdentity for TcpEndpoint {
     }
 }
 
-impl Into<EndpointLabels> for TcpEndpoint {
-    fn into(self) -> EndpointLabels {
-        use linkerd2_app_core::metric_labels::{Direction, TlsId};
-        EndpointLabels {
-            direction: Direction::Out,
-            tls_id: self.identity.as_ref().map(|id| TlsId::ServerId(id.clone())),
-            authority: None,
-            labels: None,
+// impl Into<EndpointLabels> for TcpEndpoint {
+//     fn into(self) -> EndpointLabels {
+//         use linkerd2_app_core::metric_labels::{Direction, TlsId};
+//         EndpointLabels {
+//             authority: None,
+//             direction: Direction::Out,
+//             labels: self.labels,
+//             tls_id: self.identity.as_ref().map(|id| TlsId::ServerId(id.clone())),
+//         }
+//     }
+// }
```

Comment:
should the commented-out code just be removed?

Reply:
There's a followup PR that enables it.

Reply:
https://github.com/linkerd/linkerd2-proxy/pull/654

Reply:
ah okay

Reply:
right, duh. sorry. my brain doesn't work.